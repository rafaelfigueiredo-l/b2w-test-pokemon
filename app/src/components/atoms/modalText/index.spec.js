import ModalText from "./";

describe("<ModalText />", () => {
  const component = <ModalText>Texto</ModalText>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly ModalText component", () => {
    expect(wrapper.find(".modalText")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly ModalText component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".modalText").children()).toHaveLength(1);
  });
});
