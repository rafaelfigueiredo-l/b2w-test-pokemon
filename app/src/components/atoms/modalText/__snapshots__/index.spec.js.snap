// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalText /> should mount correctly ModalText component 1`] = `
<styled.h1
  className="modalText"
>
  <h1
    className="sc-AxjAm jWlVxK modalText"
  >
    Texto
  </h1>
</styled.h1>
`;
